decaffeinate.sh

```diff
@@ -0,0 +1,45 @@
+set -ex
+
+npx bulk-decaffeinate convert --dir app/coffee
+
+npx bulk-decaffeinate clean
+
+git mv app/coffee app/js
+
+git commit -m "Rename app/coffee dir to app/js"
+
+npx prettier-eslint 'app/js/**/*.js' --write
+
+git add .
+git commit -m "Prettier: convert app/js decaffeinated files to Prettier format"
+
+npx bulk-decaffeinate convert --dir test/acceptance/coffee
+
+npx bulk-decaffeinate clean
+
+git mv test/acceptance/coffee test/acceptance/js
+
+git commit -m "Rename test/acceptance/coffee to test/acceptance/js"
+
+npx prettier-eslint 'test/acceptance/js/**/*.js' --write
+
+git add .
+git commit -m "Prettier: convert test/acceptance decaffeinated files to Prettier format"
+
+git mv app.coffee app.js
+git mv Gruntfile.coffe Gruntfile.js
+git mv config/settings.defaults.coffee config/settings.defaults.js
```

Comment:
I might need confirmation from someone knows more about our settings code (cc @jpallen) but does it gracefully handle JS files? Or do they have to be coffee?

Reply:
Looking at analytics as prior art, js is fine: https://github.com/sharelatex/analytics-sharelatex/blob/master/config/settings.defaults.js